spirv_glsl.cpp

```diff
@@ -666,6 +666,13 @@ void CompilerGLSL::emit_header()
 
 	for (auto &ext : forced_extensions)
 	{
+		if (ext == "GL_KHR_shader_subgroup")
+		{
+			declared_KHR_shader_subgroup_ballot = true;
+			declared_KHR_shader_subgroup_basic = true;
+			declared_KHR_shader_subgroup_vote = true;
```

Comment:
explain?

Reply:
GL_KHR_shader_subgroup implies all of them

Reply:
but there's no point tracking which were declared, since we need to track the reachable opcodes (parts of extensions used) because the problems with replicating functionality

Reply:
we discussed it here https://github.com/KhronosGroup/SPIRV-Cross/pull/1476#discussion_r498804887
spirv-cross already does it